packages/appcues-custom-previewer/android/gradle.properties

```diff
@@ -0,0 +1,5 @@
+AppcuesCustomPreviewer_kotlinVersion=1.7.0
+AppcuesCustomPreviewer_minSdkVersion=21
+AppcuesCustomPreviewer_targetSdkVersion=31
+AppcuesCustomPreviewer_compileSdkVersion=31
```

Comment:
normally would need 34 here to include appcues, but maybe it all just works if it inherits from the host app?

Reply:
Updated to 34. Not sure how it actually works 🤷